### PR TITLE
feat: add a nodejs binary for transmux via command line

### PR DESCRIPTION
**Pull request by brandonocasey**

### CODE REVIEW

File: bin/transmux.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  for (let i = 0; i < args.length; i++) {
+    const arg = args[i];
+
+    if ((/^--version|-v$/).test(arg)) {
+      console.log(`transmux.js v${version}`);
+      process.exit(0);
+    } else if ((/^--help|-h$/).test(arg)) {
+      showHelp();
+      process.exit(0);
+    } else if ((/^--debugger|-d$/).test(arg)) {
```
Comment (gkatsev) [inline bin/transmux.js:38]: we currently error out if `muxjs-transmux` is run without arguments, we probably should default to `showHelp()`

**Review by gkatsev — COMMENTED** [review]
Seems to create mp4 files but the timing is all messed up. Is it because there's no associated init segment?

**Comment by brandonocasey** [comment]
The timing issues could be because I set base media decode time to a default value of 100000

**Comment by brandonocasey** [comment]
Actually I think you mean the duration of the file when playing back? That happens because we set duration to `0xffffffff` by default in our generated fmp4 files.

**Comment by gkatsev** [comment]
Yeah, of the generated mp4 file.

**Review by gkatsev — APPROVED** [review]
I think I'm still getting weird durations in the output but I think we can fix it later.

**Pull request merged**